### DISCUSSION TITLE
Contributor License Agreement

### DISCUSSION THREAD

**Discussion by kkato** [cloudnative-pg/cloudnative-pg#433] [Q&A] [answered]
Hi,
I am interested in contributing to the project.
Do I need Contributor License Agreement to submit patches?

**Comment by gbartolini** [comment, marked as answer]
Nope. Our DCO is enough. See https://github.com/cloudnative-pg/cloudnative-pg/blob/main/contribute/README.md#sign-your-work

Thanks, looking forward to your contributions!
